CONTRIBUTORS.md

```diff
@@ -0,0 +1,6 @@
+# CRIPT DEVELOPMENT TEAM
```

Comment:
do you think there's a way to eventually automate this so that it grabs the contribution list from our GitHub repo or something?